cosmo_tester/framework/cfy_helper.py

```diff
@@ -97,6 +101,34 @@ def bootstrap(self,
                 task_retry_interval=task_retry_interval,
                 verbose=verbose,
                 debug=debug).wait()
+            self.upload_plugins(cfy)
```

Comment:
if they are bootstrapping on another machine, how are they calling this to begin with?
shouldn't it be some ssh/winrm call with explicit `cfy bootstrap ...`?